with_ssr/app/[movieId]/@modals/edit/page.js

```diff
@@ -0,0 +1,11 @@
+import { Dialog, MovieForm } from "@/src/entities";
+import { getSelectedMovieDetails } from "../../page";
+
+export default async function NewMovie({params}) {
```

Comment:
let's avoid export default 
```suggestion
export async function NewMovie({params}) {
```

Reply:
done